prow/config.yaml

```diff
@@ -5231,6 +5231,84 @@ postsubmits:
             requests:
               memory: "6Gi"
 
+  - name: ci-kubernetes-bazel-build-1-10
+    agent: kubernetes
+    branches:
+    - release-1.10
+    labels:
+      preset-service-account: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/bazelbuild:v20180201-0184a54dc-0.8.1
+        args:
+        - "--clean"
+        - "--git-cache=/root/.cache/git"
+        - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://kubernetes-jenkins/logs"
+        - "--" # end bootstrap args, scenario args below
+        - "--build=//... -//vendor/..."
+        - "--release=//build/release-tars"
+        env:
+        - name: TEST_TMPDIR
+          value: /root/.cache/bazel
+        # Bazel needs privileged mode in order to sandbox builds.
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - name: cache-ssd
+          mountPath: /root/.cache
+        ports:
+        - containerPort: 9999
+          hostPort: 9999
+        resources:
+          requests:
+            memory: "6Gi"
+      volumes:
+      - name: cache-ssd
+        hostPath:
+          path: /mnt/disks/ssd0
+    run_after_success:
```

Comment:
we should probably not run_after_success this

Reply:
I was following the pattern used by the release-1.9 jobs. Should this be listed as its own job instead?

Reply:
yeah this should be it's own job, they're pretty diverged. we should probably also do this to 1.9. the only "legitimate" `run_after_success` jobs left AFAIK are kubeadm related and I'm planning to fix that soon.